### PR TITLE
XS✔ ◾ Update rule.md

### PR DESCRIPTION
**Pull request by Ben0189**
Add reason of not using throw ex

## Reason for change (Issue, Email, conversation + reason, etc)

Missing reason of not using throw ex

### CODE REVIEW

File: rules/do-you-catch-and-re-throw-exceptions-properly/rule.md [hunk 1]
```diff
@@ -1,68 +1,73 @@
...
+```cs
+catch (SomeException ex) { throw ex; } 
+
+catch (SomeException ex) { someMethod(); throw ex; } 
+```
+::: bad
+Bad Example - Never re-throw exceptions by passing the original exception object. Wrap the exception or use throw.
+:::
+
+The "Throw(ex)" method resets the stack trace, causing the error to appear as if it originated from the line where "Throw(ex)" is used.
```
Comment (christoment) [inline rules/do-you-catch-and-re-throw-exceptions-properly/rule.md:47]: Suggesting more general statement and the pain point 😄 

```suggestion
Using `throw ex` resets the stack trace, obscuring the original the error and may hide highly valuable information to debug this exception.

```

File: rules/do-you-catch-and-re-throw-exceptions-properly/rule.md [hunk 2]
```diff
@@ -1,68 +1,73 @@
...
+
+```cs
+catch (SomeException) 
+{ 
+     someMethod(); 
+     throw; 
+}
+```
+
+The "Throw" method throws the current exception without altering the stack trace, allowing you to preserve information about the original exception, which is the desired behavior.
```
Comment (christoment) [inline rules/do-you-catch-and-re-throw-exceptions-properly/rule.md:57]: I think this part is unnecessary since the sentence above is already enough

Comment (Ben0189) [reply rules/do-you-catch-and-re-throw-exceptions-properly/rule.md]: Done removed

File: rules/do-you-catch-and-re-throw-exceptions-properly/rule.md (hunk 1 above)
Comment (Ben0189) [reply rules/do-you-catch-and-re-throw-exceptions-properly/rule.md]: Accepted changes

**Review by christoment — APPROVED** [review]

**Pull request merged**